execute.c

```diff
@@ -339,8 +339,8 @@ update_environment_file(char *host_name, char *socket_path, Label *host_label, i
 	close(fd);
 
 	snprintf(cmd, PATH_MAX,
-	    "renv %s %s | ssh -q -S %s %s 'cat > " REMOTE_TMP_PATH "/final.env'",
-	    op.environment_file, tmp_src, socket_path, host_name, http_port);
+	    "renv %s %s | ssh -q -S %s %s 'cat > " REMOTE_TMP_PATH "/final.env; touch " REMOTE_TMP_PATH "/local.env'",
```

Comment:
Touching `local.env` is just to ensure it can be sourced. The idea is that `renv` will append entries to it. This example sets `NEW_DB` to be checked later

```sh
mysql_init:
    [ -d /var/mysql ] && exit
    /usr/local/bin/mysql_install_db > /tmp/mysql_install.out && $SD/renv NEW_DB=yes
    rcctl start mysqld

mysql_create:
    [ "$NEW_DB" == "yes" ] && mysql <<-SQL
      CREATE DATABASE wordpress;
      CREATE USER 'wordpress'@'localhost' IDENTIFIED BY '*******';
      GRANT ALL PRIVILEGES ON * . * TO 'wordpress'@'localhost';
      FLUSH PRIVILEGES;
    SQL
```